akka-http-core/src/main/scala/akka/http/javadsl/settings/ConnectionPoolSettings.scala

```diff
@@ -58,6 +59,7 @@ abstract class ConnectionPoolSettings private[akka] () { self: ConnectionPoolSet
   @ApiMayChange
   def withResponseEntitySubscriptionTimeout(newValue: Duration): ConnectionPoolSettings = self.copy(responseEntitySubscriptionTimeout = newValue)
 
+  @deprecated("Deprecated as transport is now retrieved from ClientConnectionSettings)", "10.0.12")
```

Comment:
The most important thing is to remove `ConnectionPoolSettingsImpl.transport`. It seems this is still missing.

Reply:
@jrudolph  Done, but I had to add MIMA filters again...